### DISCUSSION TITLE
Any way to hightlight an active tree?

### DISCUSSION THREAD

**Discussion by id259** [piroor/treestyletab#3227] [Q&A] [answered]
Is there any way to change background color of tabs in an active tree? I have a problem when I hit the final indention level I get lost in trees since there is no way to know when the active tree ends and the parent tree continues.

**Comment by piroor** [comment, marked as answer]
It is impossible because TST itself doesn't provide enough hint to highlight active tree. It needs to be done by a helper extension like [TST Indent Line](https://addons.mozilla.org/firefox/addon/tst-indent-line/) developed for the purpose.

**Reply by id259** [reply]
Unfortunately, [TST Indent Line](https://addons.mozilla.org/firefox/addon/tst-indent-line/) doesn't show the line if there is no indentation, making it useless in my case. Any chance of implementing some sort of a hint so the color could be set via extra styles?

**Reply by piroor** [reply]
I meant that you can develop a helper addon which update style sheet of tabs dynamically, based on codes of TST Indent Line. Here is the logic to collect id of tabs in the active tree: https://github.com/piroor/tst-indent-line/blob/8db0c808f9e29d4106b7112eabefd272a8385150/background/background.js#L362 . The feature you expected will be there if you set background color or something extra style rules for tabs in the active tree. (Of course you can develop similar addon from scratch without my codes. I just thought that TST Indent Line is a good baseline to fork but it is not the only one solution.)

**Reply by id259** [reply]
Understood, thank you.
